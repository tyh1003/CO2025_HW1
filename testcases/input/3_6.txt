11
86 20 54 95 43 1 47 4 64 82 51
