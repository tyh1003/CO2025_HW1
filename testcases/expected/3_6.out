1 4 20 43 47 51 54 64 82 86 95 
